### PR TITLE
N115 feature db users

### PR DESCRIPTION
**Pull request by dan933**
Started work on the database work in progress data type for UserIdentifier will need to change later on down the road.

### CODE REVIEW

File: nba-api-dotnet/appsettings.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
     "EveDesktopDB": "Data Source=(localdb)\\Project_DB;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
 
     "RikiLaptopDB":"Data Source=RIKIPC;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
     "WadeLaptopDB": "Data Source=LAPTOP-98R9H465;Initial Catalog=NBA;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False",
-    "AzureDatabase": "Server=tcp:nba-instance-one.database.windows.net,1433;Initial Catalog=NBA;Persist Security Info=False;User ID=nbaAdmin;Password=nbaPrediction_1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;",
+    "AzureDatabase": "1Server=tcp:nba-instance-one.database.windows.net,1433;Initial Catalog=NBA;Persist Security Info=False;User ID=nbaAdmin;Password=nbaPrediction_1;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;",
```
Comment (EveJoyceStudent) [inline nba-api-dotnet/appsettings.json]: do we really want to include this?

File: nba-front-end/src/components/Default.tsx [hunk 2]
```diff
@@ -8,6 +8,8 @@ const linkStyle = {
 };
 
 function DefaultPage() {
+
+  console.log(process.env);
```
Comment (EveJoyceStudent) [inline nba-front-end/src/components/Default.tsx]: don't need this

File: nba-api-dotnet/appsettings.json (hunk 1 above)
Comment (dan933) [reply nba-api-dotnet/appsettings.json]: good pick up this was testing an environment variables on the server I will remove the 1

File: nba-front-end/src/components/Default.tsx (hunk 2 above)
Comment (dan933) [reply nba-front-end/src/components/Default.tsx]: ok no worries I will remove it now

**Review by EveJoyceStudent — APPROVED** [review]
looks good

**Pull request merged**